.changeset/use_consistent_curly_braces_add_forbidden_chars_check.md

```diff
@@ -0,0 +1,17 @@
+---
+"@biomejs/biome": patch
+---
+
+
+Fixes [`useConsistentCurlyBraces breaks react/no-unescaped-entities rule`](https://github.com/biomejs/biome/issues/5391)
+
+Adds an additonal check for forbidden characters: ['>', '"', ''', '}'].
+If any of these characters are detected, curly braces will be preserved.
```

Comment:
```suggestion
Fixed [`useConsistentCurlyBraces breaks react/no-unescaped-entities rule`](https://github.com/biomejs/biome/issues/5391)

Added a check for forbidden characters: `>`, `"`, `'` and `}`.
If any of these characters are detected, curly braces will be preserved.
```

Reminder that we have guidelines for the changeset: https://github.com/biomejs/biome/blob/main/CONTRIBUTING.md#writing-a-changeset

We use the past